<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Futura Md BT</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sériový monitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Futura Md BT</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sériový monitor:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="serial_monitor">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="serial_log_content">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>339</width>
        <height>214</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="console_text"/>
     </item>
     <item>
      <widget class="QPushButton" name="send">
       <property name="text">
        <string>Odeslat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
